### ISSUE TITLE
[BUG] Hover-effekt på Kirby-List

### ISSUE THREAD

**Issue by Lars-Bruun** [kirbydesign/designsystem#2830]
## Describe the bug
Hover-effekt fungerer ikke altid på list items i Kirby-list i Safari og Chrome til MacOS. Når man scroller med musen virker hover effekt dog tilsyneladende på alle items.

## Describe how to reproduce the bug

1. Go to 'Liste med mange items (så mange at der er scroll på siden). Fx Kontooversigt på 1210902925 i Jyske Bank, staging.
2. Click on 'Fold evt. kontolisten ud'
3. Move mouse cursor over list item
4. See error -->

## Which Kirby version was used?
8? 

## What was the expected behavior?
Hover-effekt på alle list-items, når cursoren bevæges over et item.

## Add any screenshots
![image](https://user-images.githubusercontent.com/69841400/216618545-75778000-2b6e-4a9e-a080-b2a352db3b7b.png)

https://user-images.githubusercontent.com/69841400/216617143-3ea7d52e-b054-48e4-82ae-4adf3fc43b67.mov

## Please complete the following information:
- MacOS
- Safari Version 16.2 (18614.3.7.1.5)

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
